libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -148,7 +148,7 @@ const AP_Param::GroupInfo AP_ICEngine::var_info[] = {
 
     // @Param: STARTCHN_MIN
     // @DisplayName: Input channel for engine start minimum PWM
-    // @Description: This is a minimum PWM value for engine start channel for an engine stop to be commanded. Setting this value will avoid RC input glitches with low PWM values from causing an unwanted engine stop. A value of zero means any PWM below 1300 triggers an engine stop.
+    // @Description: This is a minimum PWM value for engine start channel for an engine stop to be commanded. Setting this value will avoid RC input glitches with low PWM values from causing an unwanted engine stop. A value below 800 will result in any PWM between 800 to 1300 triggers an engine stop, and anything below 800 does not change the current engine state.
```

Comment:
```suggestion
    // @Description: This is a minimum PWM value on the engine start channel for an engine stop to be commanded. Setting this value will avoid RC input glitches with PWM values above this value from causing an unwanted engine stop. Setting this value below 800 will result in any PWM on the engine start channel between 800 to 1300 triggering an engine stop, but engine start channel PWMs below 800 do not change the current engine state.
```
I think this is clearer....btw this is really funky behavior...

Reply:
The suggested change isn't correct. It's glitches below the parameter it stops.

I also found it to be surprising, which is why I sent this in to document it.

Reply:
@WickedShell I think this revision completely specifies the operation now